odltools/common/constants.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2018 Red Hat Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
```

Comment:
Well odl_client has nothing to do with monitoring, it is just used by monitor.  But it could be used by many tools.  The constants file...you could make an argument to have a constants in monitor folder, but we should have a generic constants as well for hte entire project and since this is the first use of that it is not overloaded or anything.  In the future if common/constants becomes too bloated we can move specific constants into their own files, but its better to have constant variables in their own file.